linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/var_password_pam_remember_control_flag.var

```diff
@@ -20,4 +20,6 @@ options:
     "sufficient": "sufficient"
     "binding": "binding"
     "ol8": "required,requisite"
+    "rhel8": "requisite,required"
```

Comment:
I would rather have a reusable option:
```suggestion
    "requisite_and_requried": "requisite,required"
```

And then other products could use it as well.

Reply:
It is a good suggestion. I like more generic variables and also it is easier to understand the intention when reading `requisite_or_required` than `rhel8`, for example. I am updating.

Reply:
Done.